robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -280,16 +280,15 @@ def fill_gift_entry_form(self,**kwargs):
         Key is field name and value is value to be entered for field """
         for key,value in kwargs.items():
             locator=npsp_lex_locators["gift_entry"]["id"].format(key)
-            element=self.selenium.get_webelement(locator)
-            type=element.get_attribute("data-qa-locator")
+            type=self.selenium.get_webelement(locator).get_attribute("data-qa-locator")
```

Comment:
Or you could also use `type = self.selenium.get_element_attribute(locator, "data-qa-locator")`. Just FYI.

Reply:
updated